vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -105,6 +105,8 @@ See the [Build and Publish a Simple Bundle](../../tutorials/create/pb/publish-si
 |`ent bundle pack`| Generate the bundle artifacts, the bundle image and the microservice images |
 |`ent bundle pack --org [organization]`| Generate the bundle artifacts and images, passing the organization name to Docker Hub |
 |`ent bundle pack --file [my-dockerfile]`| Use a custom Dockerfile for the bundle |
+|`ent bundle pack --skip-docker-build`| Skip the building of Docker images |
```

Comment:
Add just "-s"? I think it would be helpful to let them know that -s works the same. Not sure if it belongs here or down below in the details.

Reply:
i was on the fence with this... we haven't been using shorthand and just recently changed all the `ent k`s to `ent kubectl`... the help text displays -s as an option so i think if they're in deep enough to need a shortcut it's readily available. we can discuss later, too